cata_test/input_data/example_data.csv

```diff
@@ -0,0 +1,2 @@
+hello,world

```

Comment:
I think you should delete the `cata_test` directory, unless it is used in the testing (and if so, it belongs in the tests directory).